src/Controller/interPageController.py

```diff
@@ -299,15 +319,35 @@ def show_patient(self, path):
         """
         Display patient data
         """
+        patient_dict_container = PatientDictContainer(
+            path,
+            self.bar_window.ext.read_data_dict,
+            self.bar_window.ext.file_names_dict,
+            rois=self.bar_window.ext.rois,
+            raw_dvh=self.bar_window.ext.raw_dvh,
+            dvh_x_y=self.bar_window.ext.dvh_x_y,
+            raw_contour=self.bar_window.ext.dict_raw_ContourData,
+            num_points=self.bar_window.ext.dict_NumPoints,
+            pixluts=self.bar_window.ext.dict_pixluts,
+        )
         # Loads the main window  by providing the necessary data obtained by the progress bar
-        self.patient_window = MainWindow(path, self.bar_window.ext.read_data_dict, self.bar_window.ext.file_names_dict,
-                                         self.bar_window.ext.rois, self.bar_window.ext.raw_dvh,
-                                         self.bar_window.ext.dvh_x_y, self.bar_window.ext.dict_raw_ContourData,
-                                         self.bar_window.ext.dict_NumPoints, self.bar_window.ext.dict_pixluts)
+        self.patient_window = MainWindow(
+            # path,
+            patient_dict_container,
```

Comment:
and then pass in just the patient_dict_container.
It would be reasonable for a review comment to require that I eliminate all of the commented out arguments.
I left them there to make it clear what the change was, and why.